middleware/bottle.js

```diff
@@ -0,0 +1,34 @@
+import Bottleneck from "bottleneck";
```

Comment:
delete the whole file if we're not using it

Reply:
done